### PR TITLE
feat: Toolbox - major version bump

### PR DESCRIPTION
**Pull request by tbwiss**
# Description

> **Note**
> 
> Please provide a description of the work completed in this PR below

Bump the Toolbox to v2.0.0

### CODE REVIEW

File: .changeset/hip-fireants-kiss.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@neo4j/graphql-toolbox": major
+---
+
+feat: Toolbox - major version bump to version 2.0.0
```
Comment (mjfwebb) [inline .changeset/hip-fireants-kiss.md:5]: Is this good enough? Do we not need to add context for why it's now 2.0.0?

Comment (tbwiss) [reply .changeset/hip-fireants-kiss.md:5]: Nah it's not 😃 . I'll extend this.

File: .changeset/hip-fireants-kiss.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+A high-level overview of changes made:
+
+- All the editors are now powered by [CodeMirror](https://codemirror.net/) version 6 and support all the features and functions from the previous version.
+- On the query editor page we've introduced tabs to store several queries and mutations. Improved the solution for resizing the different editors (query, variables, and response).
+- The favorites on the type definitions page did get an update. It's now possible to order favorites and download them individually.
```
Comment (mjfwebb) [inline .changeset/hip-fireants-kiss.md:11]:
```suggestion
- The favorites feature on the type definitions page has been updated. It's now possible to order favorites and download them individually.
```

**Review by mjfwebb — APPROVED** [review]
LGTM! Thanks :)

**Pull request merged**